docs/scripts/build-component-props.mjs

```diff
@@ -13,6 +13,7 @@ const parser = reactDocgenTypescript.withCustomConfig('./tsconfig.json', {
     'SelectComponent',
     'ComboBoxComponent',
     'RadioComponent',
+    'SelectListComponent',
```

Comment:
What does this do?

Reply:
this is because this are compound components with the collection item thing i think and thats why they have to be in this, I can explain it tomorrow to you :) HArd to explain for me